.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Please include the following information when opening an issue:
```

Comment:
Is this a template? Shouldn't it rather contain headlines for each step that people should fill in?